docs/api.md

```diff
@@ -501,10 +501,10 @@ $ bower install --config.interactive=false
 Bower works by default in interactive mode. There are few ways of disabling it:
 
 - passing `CI=true` in environment
-- passing `--config.interactive=false` to bower command
-- attaching a pipe to bower (e.g. `bower install | cat`)
+- passing `--config.interactive=false` to Bower command
```

Comment:
"passing" is strange terminology. Perhaps: 

* Setting `CI=true` as an environment variable
* Adding `--config.interactive=false` to any `bower` command (e.g. `bower install jquery --config.interactive=false`)